Src/SwqlStudio/MainForm.cs

```diff
@@ -17,7 +18,7 @@ namespace SwqlStudio
         UseSynchronizationContext = false)]
     internal partial class MainForm : Form, IApplicationService
     {
-        private static readonly SolarWinds.Logging.Log log = new SolarWinds.Logging.Log();
+        private static readonly ILogger<MainForm> log = Program.LoggerFactory.CreateLogger<MainForm>();

```

Comment:
The absence of true dependency injection makes these initializations a bit uglier than they would otherwise be.